checker/tests/nullness/java8inference/Issue402.java

```diff
@@ -3,13 +3,14 @@
 
 // @below-java8-jdk-skip-test
 
-// This test now seems to fail because of Issue 1006
-// @skip-test
-
 import java.util.Comparator;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
+@SuppressWarnings({
+    "nullness",
+    "keyfor"
+}) // Type argument inference does not infer the correct types.
```

Comment:
The suggestion about issue number is a good one.

Our standard is to put the `@SuppressWarnings` explanation on the same line with the `@SuppressWarnings` annotation; in this case, google-code-format has broken the lines, but I wouldn't move the annotations to accommodate the formatting tool.